_includes/releases/v23.1/v23.1.2.md

```diff
@@ -1,24 +1,209 @@
 ## v23.1.2
 
-Release Date: May 25, 2023
+Release Date: May 30, 2023
 
 {% include releases/release-downloads-docker-image.md release=include.release %}
 
-{{site.data.alerts.callout_info}}
-This is an incomplete list of the changes included in the v23.1.2 release. The full list will be published soon.
-{{site.data.alerts.end}}
+<h3 id="v23-1-2-backward-incompatible-changes">Backward-incompatible changes</h3>
+
+- Schema names in the [`search_path`](../v23.1/sql-name-resolution.html#search-path) now respect case and must be quoted with double quotes. Previously, if a `search_path` was specified in the connection string parameters, it would be treated case sensitive always. [#101493][#101493]
```

Comment:
Hi @rafiss, would love your input on this PR that you worked on - did you mean to say that initially `search_path` was case insensitive?

Reply:
Got clarification from the engineer:

`search_path` is an existing session variable. the previous behavior was:
if you specified it with `SET search_path = xyz`, then the schema names would be treated case-insensitive by default. but you could use double quotes to make the names be recognized as mixed-case
if you specified `search_path` as part of the connection string, then the schema names would always be handled as a mixed-case name, even if you didn't use double quotes. and if the value did have double quotes in it, the quotes themselves would be treated as part of the name
the new behavior is that `search_path` is always handled the first way i described

Reply:
Would it make sense to say this? @MichaelTrestman 

Schema names in the [`search_path`](../v23.1/sql-name-resolution.html#search-path) session variable now respect case and must be quoted with double quotes. Previously, if a `search_path` was specified in the connection string parameters, it would be treated case insensitive by default. [#101493][#101493]